src/test/ui/issues/issue-31109.rs

```diff
@@ -1,6 +0,0 @@
-fn main() {
-    // FIXME(#31407) this error should go away, but in the meantime we test that it
-    // is accompanied by a somewhat useful error message.
-    let _: f64 = 1234567890123456789012345678901234567890e-340;
-    //~^ ERROR could not evaluate float literal (see issue #31407)
```

Comment:
Maybe keep the tests and make them expect compilation to succeed?

Reply:
@bjorn3 I don't see why - see my comments in https://github.com/rust-lang/rust/pull/86761#discussion_r663414379

Reply:
I originally implemented such tests, but since everything relies on `dec2flt` anyway, the checks would be redundant, and would require building a stage 1 compiler to test them, so at the recommendation of others, I removed them.